AnkiDroid/src/main/java/com/ichi2/anki/MyAccount.java

```diff
@@ -176,6 +179,25 @@ private void initAllContentViews() {
         mUsername = mLoginToMyAccountView.findViewById(R.id.username);
         mPassword = mLoginToMyAccountView.findViewById(R.id.password);
         mPasswordLayout = mLoginToMyAccountView.findViewById(R.id.password_layout);
+        ImageView mAnkidroidLogo = mLoginToMyAccountView.findViewById(R.id.ankidroid_logo);
+
+        //checking if device is in horizontal mode or not .
+        OrientationEventListener myOrientationEventListener = new OrientationEventListener(this,SensorManager.SENSOR_DELAY_NORMAL)
+        {
+            @Override
+            public void onOrientationChanged(int orientation)
+            {
+                boolean PORTRAIT_MODE = ((orientation < 100) || (orientation > 280));
+                // if device is in horizontal mode then screen might not have enough space for ankidroid logo
+                // so we will invisible logo for horizontal mode only
+                if (PORTRAIT_MODE) {
+                    mAnkidroidLogo.setVisibility(View.GONE);
+                } else {
+                    mAnkidroidLogo.setVisibility(View.VISIBLE);
+                }
```

Comment:
```suggestion
                // if device is in horizontal mode then screen might not have enough space for ankidroid logo
                // so we will invisible logo for horizontal mode only
                if ((orientation < 100) || (orientation > 280)) {
                    mAnkidroidLogo.setVisibility(View.GONE);
                } else {
                    mAnkidroidLogo.setVisibility(View.VISIBLE);
                }
```

If the variable is only used in one spot, the temp variable is not needed